MTM1M3Movements.py

```diff
@@ -91,8 +91,14 @@ class MTM1M3Movements(MTM1M3Test):
 
     POSITION_TOLERANCE = 8 * POS_TOL_UNIT
     ROTATION_TOLERANCE = 1.45 * ROT_TOL_UNIT
-    LOAD_PATH_FORCE = 0.0
-    LOAD_PATH_TOLERANCE = 100.0
+
+    # Base force, moments and their tolerances were
+    # measured on the mirror. Adjust as needed.
+    LOAD_PATH_FORCE = [275, 55, 1950] * u.N
+    LOAD_PATH_FORCE_TOLERANCE = [200, 200, 200] * u.N
+    LOAD_PATH_MOMENTS = [1000, 232, 480] * u.N * u.m
+    LOAD_PATH_MOMENTS_TOLERANCE = [200, 200, 200] * u.N * u.m
```

Comment:
In this case, why not just use a configuration file instead? In addition, it looks like the elements are x, y, and z direction. If this is correct, please consider to document this.

Reply:
This is a simple test. Configuration file will just add complexity without any benefits - it must be stored in some (constant) path (so one can run the script from another directory). This is designed so far to run only on L3, in zenith pointing. It isn't designed to be run on TMA at some elevation angle. It's fair enough to expect end user to edit the script, if baseline changes (that's expected with glass mirror installation).